src/base/static/state/ducks/datasets-config.js

```diff
@@ -0,0 +1,49 @@
+// Selectors:
+export const datasetSlugsSelector = state =>
+  state.datasetsConfig.map(config => config.dataset_slug);
+
+export const hasAnonAbilityInAnyDataset = ({ state, ability, submissionSet }) =>
+  state.datasetsConfig.some(config =>
+    config.anonymous_permissions.some(
+      perm =>
+        (perm.submission_set === "*" ||
+          perm.submission_set === submissionSet) &&
+        perm.abilities.includes(ability),
+    ),
+  );
+export const hasAnonAbilityInDataset = ({
```

Comment:
See my comment above - we can delete this function and call `hasAnonAbilityInAnyDataset` by passing in an array of `datasetSlugs`.

We can also rename `hasAnonAbilityInAnyDataset` to `hasAnonAbilityInDatasets`